src/meshio/netgen/_netgen.py

```
@@ -0,0 +1,350 @@
+    pi[:, col1:col2] = block.data[:, pmap] + 1
+    pi[:, col2:] = np.repeat([post_data], len(block), axis=0)
+    if index is not None:
+        pi[:, i_index] = index
+    np.savetxt(f, pi, "%i")
+
+
+def _skip_block(f):
+    n = int(f.readline())
+    for i in range(n):
+        f.readline()
+
+
+def read_buffer(f):
+    points = []
+    cells = []
+    cells_index = []
+
+    while True:
+        # fast-forward over blank lines
+        line, is_eof = _fast_forward_over_blank_lines(f)
+        if is_eof:
+            break
+
+        line = line.strip()
+        if line.startswith("#"):
+            continue
+        elif line == "dimension":
+            dimension = int(f.readline())
+        elif line == "geomtype":
+            f.readline()
+        elif line == "mesh3d":
+            continue
+        elif line == "points":
+            npoints = int(f.readline())
+            for i in range(npoints):
+                p = filter(None, f.readline().strip().split(" "))
+                points.append(list(map(float, p)))
+            points = np.array(points)
+            if dimension == 2:
+                points = points[:, :2]
+
+        elif line in [
+            "pointelements",
+            "edgesegments",
+            "edgesegmentsgi",
+            "edgesegmentsgi2",
+            "surfaceelements",
+            "volumeelements",
+        ]:
+            _read_cells(f, line, cells, cells_index)
+
+        elif line == "endmesh":
+            break
+
+        elif line in [
+            "bcnames",
+            "cd2names",
+            "cd3names",
+            "face_colours",
+            "identifications",
+            "identificationtypes",
+            "materials",
+            "singular_edge_left",
+            "singular_edge_right",
+            "singular_face_inside",
+            "singular_face_outside",
+            "singular_points",
+        ]:
+            _skip_block(f)
+        else:
+            continue
```

Why not `raise RuntimeError("Unknown environment")`?